Formula/moarvm.rb

```diff
@@ -0,0 +1,37 @@
+class Moarvm < Formula
+  desc "VM for NQP And Rakudo Perl 6"
+  homepage "https://moarvm.org"
+  url "https://moarvm.org/releases/MoarVM-2018.12.tar.gz"
+  sha256 "e433a2a56953fca16ed8eaf9a597f25a7e1556436888609429f38529c241693b"
+  head "https://github.com/MoarVM/MoarVM.git"
+
+  depends_on "perl" => :build
+  depends_on "pkg-config" => :build
+  depends_on "libatomic_ops"
+  depends_on "libffi"
+  depends_on "libtommath"
+  depends_on "libuv"
+
+  def install
+    libffi = Formula["libffi"]
+    ENV.remove "CPPFLAGS", "-I#{libffi.include}"
```

Comment:
This shouldn't be necessary, a non-existing directory should not be an issue in `CPPFLAGS`.